src/tzdata/archive.jl

```diff
@@ -1,7 +1,9 @@
 import Compat: @static, Sys, devnull
 
-if Sys.iswindows()
-    const exe7z = joinpath(Sys.BINDIR, "7z.exe")
+function __init__()
+    if Sys.iswindows()
+        const exe7z = joinpath(Sys.BINDIR, "7z.exe")
```

Comment:
`const` only makes sense in the global scope currently. Also we need this variable available as a global so this should just be changed to `global`